### DISCUSSION TITLE
new Release planned?

### DISCUSSION THREAD

**Discussion by jnu144** [canopen-python/canopen#293] [General]
Hi @christiansandberg 

Is it possible to create in near future a new version from the current master branch and also make it available from PyPi?

**Comment by christiansandberg** [comment]
I've been meaning to do that for a while. I have now published [2.0.0b1](https://pypi.org/project/canopen/2.0.0b1/) to PyPI so it is easier to test it out. I'll try to finish the release notes ASAP.
